### ISSUE TITLE
umzug.up() not working with sequelize 5

### ISSUE THREAD

**Issue by toymachiner62** [sequelize/umzug#191]
I have a project that is using umzug for migrations. Everything is 🍑y with sequelize `4.42.x`, but when I upgrade to version `5.3.1`, I'm now getting the error

```
{ SequelizeConnectionAcquireTimeoutError: Operation timeout
    at pool.acquire.catch.err (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:281:53)
    at tryCatcher (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/util.js:16:23)
    at /Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/catch_filter.js:17:41
    at tryCatcher (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/promise.js:690:18)
    at _drainQueueStep (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
  name: 'SequelizeConnectionAcquireTimeoutError',
  parent:
   TimeoutError: Operation timeout
       at Timeout._timeout.setTimeout (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/sequelize-pool/lib/Deferred.js:19:19)
       at ontimeout (timers.js:436:11)
       at tryOnTimeout (timers.js:300:5)
       at listOnTimeout (timers.js:263:5)
       at Timer.processTimers (timers.js:223:10),
  original:
   TimeoutError: Operation timeout
       at Timeout._timeout.setTimeout (/Users/tomcaflisch/Sites/ciitizen/ciitizen-profile-service/node_modules/sequelize-pool/lib/Deferred.js:19:19)
       at ontimeout (timers.js:436:11)
       at tryOnTimeout (timers.js:300:5)
       at listOnTimeout (timers.js:263:5)
       at Timer.processTimers (timers.js:223:10) }
```

Here's my code:

```js
// db is an instance of sequelize
async function runMigrations (db) {
  const cfg = {
    storage: 'sequelize',
    storageOptions: {
      sequelize: db
    },

    // see: https://github.com/sequelize/umzug/issues/17
    migrations: {
      params: [
        db.getQueryInterface(), // queryInterface
        db.constructor, // DataTypes
        function () {
          throw new Error('Migration tried to use old style "done" callback. Please upgrade to "umzug" and return a promise instead.')
        }
      ],
      path: path.join(__dirname, 'migrations'),
      pattern: /\.js$/
    }
  }
  const umzug = new Umzug(cfg)
  return umzug.up()
}
```
Any ideas why? Is there anything I need to change as part of the sequelize 4 -> 5 upgrade?

**Comment by knoid** [comment]
I'm using sequelize and postgres and got the same problem but in my case it didn't timeout, it just went on and on.

**Comment by knoid** [comment]
Got fixed for me on latest version: 5.8.12
